integration/README.md

```diff
@@ -0,0 +1,7 @@
+# OpenLineage Integrations
+|Integration|Release Link|Compatible Version|Latest Release|
+|-|-|-|-|
+|Apache Airflow|https://github.com/apache/airflow/releases|1.10.12+, 2.1+ (experimental)|2.2.5|
+|dagster|https://github.com/dagster-io/dagster/releases|0.13.8+|0.14.7|
+|dbt|https://github.com/dbt-labs/dbt-core/releases|0.20+|1.0.4|
+|spark|https://github.com/apache/spark/tags|2.4.6, 3.1.2, 3.2.1+|3.2.1|
```

Comment:
Minor style points - 

* Drop the "Apache" prefix for Airflow or add it to the Spark integration (they're both Apache projects and should be treated the same). 
* Capitalization should be consistent across the integrations. Let's not capitalize Airflow, but lowercase the others. For consistency with the actual project names, I'd capitalize all of them except dbt, which is always lowercase at https://www.getdbt.com/